### DISCUSSION TITLE
Constributions Activity Problems

### DISCUSSION THREAD

**Discussion by diffcreate** [community/community#83926] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/112700046/da518987-d112-4dac-bc6a-d12048218a05)
So I have been commiting a lot but some of it it doesn't showing as a constributions activity in my profile, did I miss something?

**Comment by MuhammadArifinHakim** [comment, marked as answer]
There could be several reasons why your contributions are not appearing on your GitHub activity graph :
1. Ensure that the email address associated with your local Git commits matches the email address associated with your GitHub account.
2. Ensure that you have pushed your commits to the remote repository on GitHub.

For my case, it's always option number 2

**Reply by diffcreate** [reply]
Ah I see, I will try it

**Reply by MuhammadArifinHakim** [reply]
you're welcome brotha
